cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -188,7 +188,7 @@ describe(
             })
 
             it('shows mod dashboard left navbar buttons with correct counts and functionality', () => {
-                //The number for include text is for the status in the mock data
+                // The number for include.text is for the status in the mock data.
```

Comment:
Were these periods manually added? I'm not sure include.text is correct

Reply:
That period was intentional, because it's reading from `include.text`. I can remove it if it's confusing though.